ansible/roles/test/files/ptftests/py3/advanced-reboot.py

```diff
@@ -1003,8 +1003,14 @@ def check_warmboot_finalizer(self, finalizer_timeout):
         self.finalizer_state = self.get_warmboot_finalizer_state()
         self.log('warmboot finalizer service state {}'.format(self.finalizer_state))
         count = 0
-        while self.finalizer_state == 'activating':
-            self.finalizer_state = self.get_warmboot_finalizer_state()
+        while self.finalizer_state != 'inactive':
```

Comment:
why do we change the logic from ==activating to !=inactive ?

Reply:
> why do we change the logic from ==activating to !=inactive ?

The is some chance that self.get_warmboot_finalizer_state() will return "" if the it some problem to access the dut